### PR TITLE
Home page Markup

### PR DESCRIPTION
**Pull request by EphraimAssi**
I did a markup for the home page following the Wire frame design.

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -3,63 +3,117 @@ import styles from '../styles/Home.module.css'
...
-          Welcome to <a href="https://nextjs.org">Next.js!</a>
-        </h1>
+      <header>
+        <figure>
+          <img src="#" alt="logo.jpg"></img>
+          <figcaption>
+            Governement of Canada
+            Gouvernement du Canada
+          </figcaption>
+        </figure>
```
Comment (pcraig3) [inline pages/index.js:19]: I would say double-check what Canada.ca uses for their header. We probably don't want a figure here, they are using `<div>`s on Canada.ca

Eg: https://design.canada.ca/mandatory-templates/home-page.html#specifications
"Working examples" links to the main canada.ca homepage: https://www.canada.ca/en.html

For the header, it looks like they are using 

```
<div>
    <a href="#" >
	<img src="..." alt="Government of Canada" property="logo">
        <span lang="fr">Government of Canada</span>
</div>
```

File: pages/index.js [hunk 2]
```diff
@@ -3,63 +3,117 @@ import styles from '../styles/Home.module.css'
...
 
-        <p className={styles.description}>
-          Get started by editing{' '}
-          <code className={styles.code}>pages/index.js</code>
-        </p>
+        <section>
```
Comment (pcraig3) [inline pages/index.js:21]: Let's do `<div>` here. Usually you want to use `<section>` in the page body, not the header element.

File: pages/index.js [hunk 3]
```diff
@@ -3,63 +3,117 @@ import styles from '../styles/Home.module.css'
...
-            <p>Learn about Next.js in an interactive course with quizzes!</p>
-          </a>
+        <section>
+          <figure>
+            <img src="#" alt="serviceCanada.jpg"></img>
+            <h1>
+              Servica Canada Labs
+              Make government work for you
+            </h1>
+            <figcaption>
```
Comment (pcraig3) [inline pages/index.js:37]: Let's make this a `<div>` too.

File: pages/index.js [hunk 4]
```diff
@@ -3,63 +3,117 @@ import styles from '../styles/Home.module.css'
...
-        <a
-          href="https://vercel.com?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Powered by{' '}
-          <img src="/vercel.svg" alt="Vercel Logo" className={styles.logo} />
-        </a>
+      <footer>
+        <figure>
```
Comment (pcraig3) [inline pages/index.js:92]: This is probably fine as a `<div>`.

**Review by pcraig3 — CHANGES_REQUESTED** [review]
Looks pretty good overall (not beautiful, but mostly we're all right here).

If you're wondering how to code something, canada.ca has most of the stuff you're looking for. Also there's the [Canada.ca Design System](https://www.canada.ca/en/government/about/design-system.html) which has details about a lot of this stuff.

File: pages/index.js (hunk 4 above)
Comment (EphraimAssi) [reply pages/index.js:92]: Perfect I will make those changes

**Review by pcraig3 — APPROVED** [review]
Thanks @EphraimAssi, this will cause a conflict with #5 (or that one with this one, depending on which is merged first), but should be easy enough to overcome.

**Pull request merged**